inference-engine/src/gna_plugin/frontend/layer_quantizer.hpp

```diff
@@ -434,7 +434,8 @@ class DataQuantizer<Desc, InferenceEngine::CNNLayer *> : public DataQuantizerBas
                 if (LayerInfo(*cnnLayer).isActivation() ||
                     LayerInfo(*cnnLayer).isCopy() ||
                     LayerInfo(*cnnLayer).isNonFunctional() ||
-                    LayerInfo(*cnnLayer).isPermute()) {
+                    LayerInfo(*cnnLayer).isPermute() ||
+                    LayerInfo(*cnnLayer).isConst()) {
```

Comment:
@esmirno Could you please comment if this is expected to have always 16bit consts?
Can this const represent i.e. biases?

Reply:
yes right now we expecting only fp16/fp32 as you may see bellow in the code other const layer are prohibited. 
In future having native I32 inputs support as bias pointer might be a new feature.  Or I8 support as weights of I8 affine layer i guess. Other integer types has to be converted somehow, like I64 - for example depending on network we still can propagate I64 down to the network using sequence of I32 elements.